### PR TITLE
[Issue #134]Document Things end point

### PR DESCRIPTION
**Pull request by isabelrios**

### CODE REVIEW

**Comment by isabelrios** [comment]
@ferjm I started documenting Things end point, if that looks good I can continue and add the rest of end points.

File: doc/API.md [hunk 1]
```diff
@@ -109,17 +109,21 @@ The currently-defined error responses are:
 ## GET /
 ### Request
 ```ssh
+curl -X GET -H "Cache-Control: no-cache" "http://localhost:8080/v1.0"
 ```
 ### Response
 ```ssh
+{"value":[{"name":"Datastreams","url":"http://localhost:8080/v1.0/Datastreams"},{"name":"FeaturesOfInterest","url":"http://localhost:8080/v1.0/FeaturesOfInterest"},{"name":"HistoricalLocations","url":"http://localhost:8080/v1.0/HistoricalLocations"},{"name":"Locations","url":"http://localhost:8080/v1.0/Locations"},{"name":"Observations","url":"http://localhost:8080/v1.0/Observations"},{"name":"ObservedProperties","url":"http://localhost:8080/v1.0/ObservedProperties"},{"name":"Sensors","url":"http://localhost:8080/v1.0/Sensors"},{"name":"Things","url":"http://localhost:8080/v1.0/Things"}]}
```
Comment (ferjm) [inline doc/API.md]: Could we JSON prettify these requests, please? http://jsonprettyprint.com/

**Comment by ferjm** [comment]
Thank you Isabel! Looks good.

**Comment by isabelrios** [comment]
@ferjm Please final review to be sure it will look fine before merging. Thanks!

File: doc/API.md [hunk 2]
```diff
@@ -109,17 +109,73 @@ The currently-defined error responses are:
...
 ```ssh
+curl -X GET -H "Cache-Control: no-cache" "http://localhost:8080/v1.0"
 ```
 ### Response
 ```ssh
+{
+  "value": [
+    {
+      "name": "Datastreams",
+      "url": "http:\/\/localhost:8080\/v1.0\/Datastreams"
```
Comment (ferjm) [inline doc/API.md]: Is there any way we could remove the escaping characters `\`?

**Pull request closed without merging**